package.json

```diff
@@ -16,31 +16,27 @@
   },
   "license": "BSD-2-Clause",
   "dependencies": {
-    "@godaddy/terminus": "^4.6.0",
-    "amoeba": "0.9.0",
-    "bunyan": "1.8.14",
-    "cookies": "0.8.0",
-    "lodash": "4.17.20",
-    "moment": "^2.29.0",
-    "mongodb": "^5.0.1",
-    "mongodb-legacy": "^5.0.0",
-    "request": "2.88.2",
-    "restify": "8.5.1",
-    "sundial": "1.6.0",
+    "@godaddy/terminus": "4.12.1",
+    "amoeba": "1.0.2",
+    "bunyan": "1.8.15",
+    "lodash": "4.17.21",
+    "moment": "2.29.4",
+    "mongodb-legacy": "6.0.0",
+    "sundial": "1.7.1",
     "tidepool-gatekeeper": "0.2.4",
```

Comment:
Will require updates to latest `tidepool-gatekeeper`, `tidepool-seagull-client`, and `user-api-client` once published.